EventCountColorCode.h

```diff
@@ -0,0 +1,46 @@
+#ifndef EVENTCOLORCODE_H
+#define EVENTCOLORCODE_H
+
+/******************************************************************************/
+/***    \file        EventCountColorCode.h
+***     \author      Pratik Jaiswal               
+***
+***     \brief       Header for EventCountColorCode.h
+***/       
+/*****************************************************************************/
+
+/*=============================================================================
+=======                            INCLUDES                             =======
+=============================================================================*/
+#include <stdio.h>
+#include <assert.h>
+  
+ /*=============================================================================
+=======                Enums for Color Code                             =======
+=============================================================================*/
+enum MajorColor {WHITE, RED, BLACK, YELLOW, VIOLET};
```

Comment:
The enum is in the .h file, while the names of the colors are in the .c file.
What mechanism would you impose to keep them in sync?